.rubocop.yml

```diff
@@ -74,10 +75,10 @@ Style/IndentationWidth:
 Style/IndentHash:
   Severity: fatal
   Enabled: true
+  EnforcedStyle: consistent
 
 Style/InlineComment:
-  Severity: fatal
-  Enabled: true
+  Enabled: false
```

Comment:
This means allow inline comments - the setting before _only_ allowed comments at the top of a class/module.